### DISCUSSION TITLE
How to sign out when session ends (tab/broswer closed)?

### DISCUSSION THREAD

**Discussion by AhsanSarwar45** [supabase/supabase#17752] [Questions]
I am trying to implement 'remember me' functionality. In case the user leaves it unchecked, I want them to be signed out after the session ends (tab/browser closed). In firebase, I could set persistence to session for such functionality, although that was for client side auth. Currently, I am using sveltekit auth helpers for server side auth using the [official guide](https://supabase.com/docs/guides/auth/auth-helpers/sveltekit). Is there any way of doing so?

**Comment by silentworks** [comment]
That would be an oddly weird behaviour for any website to have. As a user I don't expect to be signed out when I close a tab/browser. I expect to be signed out when my session expires (the JWT Supabase uses expires after 1 hour). Also a `remember me` functionality won't work beyond one week as that's the longest you can set a JWT expiry to (It's not recommended to do this).

**Comment by james-benns** [comment]
I don't think this is unusual behaviour - anyway the functionality already is already there, it just seems there is no way to toggle it. It would be great to have a setAuthPersistence method or something.
